doc/howto/autoinstall-quickstart.rst

```diff
@@ -1,48 +1,39 @@
-.. _autoinstall_quickstart:
+.. _autoinstall_quick-start:
 
 Autoinstall quick start
-***********************
+=======================
 
-The intent of this page is to provide simple instructions to perform an
-autoinstall in a VM on your machine.
+This guide provides instructions to use autoinstall with a current version of Ubuntu for the amd64 architecture in a virtual machine (VM) on your computer.
 
-This page assumes that you are installing a recent Ubuntu release. However,
-for older releases, you can substitute the name of the ISO image but the
-instructions should otherwise be the same.
-
-This page also assumes you are on the AMD64 architecture. There is a
-:ref:`version for s390x<autoinstall-quickstart-s390x>` too.
+For older Ubuntu releases, substitute the version in the name of the ISO image. The instructions should otherwise be the same. See :ref:`<autoinstall-quick-start-s390x>` for instructions on installing on the s390x architecture.
```

Comment:
Actually, this ref isn't generating a link either. Is this just an side effect of the build site?